game-core/src/main/java/games/strategy/triplea/delegate/battle/UnitBattleComparator.java

```diff
@@ -48,7 +47,7 @@ public UnitBattleComparator(
       final Collection<TerritoryEffect> territoryEffects,
       final GameData data,
       final boolean bonus) {
-    this(defending, costs, territoryEffects, data, bonus, false, false);
+    this(defending, costs, territoryEffects, data, bonus, false);
   }
 
   public UnitBattleComparator(
```

Comment:
Complete side note, I think we got an error report stating that this comparator is invalid. The error message was "contract violated". If the following are units, { u1, u2, u3}, and if {u1 < u2}, and {u2 < u3}, then the following should be true {u1 < u3}. Apparently it was not and we got an exception for that.